integrationTests/singleShard/transaction/interceptedResolvedTx_test.go

```diff
@@ -20,65 +20,52 @@ func TestNode_RequestInterceptTransactionWithMessenger(t *testing.T) {
 		t.Skip("this is not a short test")
 	}
 
-	dPoolRequester := createTestDataPool()
-	dPoolResolver := createTestDataPool()
-
-	shardCoordinator := &sharding.OneShardCoordinator{}
-
 	fmt.Println("Requester:")
-	nRequester, mesRequester, sk1, resolversFinder := createNetNode(
-		dPoolRequester,
-		createAccountsDB(),
-		shardCoordinator)
+	nRequester := integrationTests.NewTestProcessorNode(1, 0, 0, "0")
```

Comment:
it would be better to construct with named variables, as it is hard to understand what 1, 0, 0, "0" means.

Reply:
Changed here and also for the other integration tests.